lib/internal/util/inspector.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const hasInspector = process.config.variables.v8_enable_inspector === 1;
+const inspector = hasInspector ? require('inspector') : undefined;
+
+function sendInspectorCommand(cb, onError) {
+  if (!hasInspector) return onError();
+  try {
+    const session = new inspector.Session();
```

Comment:
Doesn't that make it impossible to use the inspector's JS API from the REPL?  Inspector sessions are like Highlander, there can be only one.

Reply:
As long as it stays disconnected between invocations of this function, I don't think keeping a Session object around is a problem.

Reply:
```
$ ./node
> const xxx = 1
undefined
> xxx # tab completion works
1
> const inspector = require('inspector')
undefined
> const session = new inspector.Session()
undefined
> session.connect()
undefined
> xx # tab completion does not work
ReferenceError: xx is not defined
> session.disconnect()
undefined
> xxx # tab completion works again
1
> 
```

Reply:
@TimothyGu I think one session instance per process is enough. I made that change.

Reply:
@targos it would probably be good to add the example you provided as test (while I would not test for not having the completion while a session is open but that the user session works as expected).